extensions/wikia/Staging/Staging.setup.php

```diff
@@ -29,6 +29,5 @@
  * @author macbre
  * @see PLATFORM-664
  */
-$wgBaseShareKeyPrefix = $wgSharedKeyPrefix;
```

Comment:
B/c this extension is only loaded on staging environments, `$wgBaseShareKeyPrefix` wasn't getting set in production which is where we need it. Moving this assignment to CommonSettings.php